recipes/gcc/all/conanfile.py

```diff
@@ -1,126 +1,112 @@
-from conans import ConanFile, tools, AutoToolsBuildEnvironment
-from conans.errors import ConanException, ConanInvalidConfiguration
 import os
 
-required_conan_version = ">=1.29.1"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.files import get, replace_in_file, rmdir, rm, copy
+from conan.tools.gnu import AutotoolsToolchain, Autotools
```

Comment:
Please add "+ conan v2" in the title !!!! this is amazing

Reply:
done